### DISCUSSION TITLE
Scheduler running Once and PreventingOverLap is Running more than Once

### DISCUSSION THREAD

**Discussion by realapak** [jamesmh/coravel#343] [General]
With the following configuration: 

`scope.ServiceProvider.UseScheduler(scheduler =>
            {
                scheduler.OnWorker(message.Text);
                scheduler.ScheduleWithParams<MessagesInvocable>(message)
                //.Cron($"{DateTime.UtcNow.Minute} {DateTime.UtcNow.Hour} {DateTime.UtcNow.Day} {DateTime.UtcNow.Month} *")
                .EverySeconds(1)
                .PreventOverlapping(message.Text)
                .Once();
            });`

And executing this for 10 messages I always have more than one execution for at least one message.

Why do you think this happens?

**Comment by jamesmh** [comment]
I think I need more details:
- Which version of Coravel are you using?
- How many separate scheduled tasks are you using?
- How many separate workers?
- Could you show me a larger sample from your configuration code?

Thank you!
